### PR TITLE
Bug 905974 - [bb][refactor] Switches

### PR DESCRIPTION
**Pull request by basiclines**
I've created 2 skin files: `skin.default.css` and `skin.check&radio.css`.
I think the best approach is to treat the `on/off` switch as the default apperance and then generate that skin file for the radio/checks ones. In that way we avoid to over-defining classnames in the elements like `bb-switch switch` or `bb-switch radio danger`.

I will like to remove some images in order to achieve a more "pure" css solution. But i'm wating to confirm it with @KevinGrandon because a possible performance impact in long element lists.

### CODE REVIEW

**Comment by basiclines** [comment]
I've pushed a new version with a more pure css solution, now we only use a `tick.png` as image for checks&radios

File: style/switches/skin.default.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/**
+ * Default skin: on/off
+ */
+
+.bb-switch input +  span {
```
Comment (rik) [inline style/switches/skin.default.css]: Nit: Extra space before `span`.

**Comment by rik** [comment]
Please don't use `&` in filenames, it's asking for trouble.

There's a lot of sizes in those files. When it's a addition of other sizes, please use `calc()` and add comments about which size it refers to. Makes it easier to tweak the dimensions and assets later.

**Comment by basiclines** [comment]
Just applied your feedback @Rik thx!

**Pull request merged**